chromium_src/chrome/installer/setup/brave_behaviors.cc

```diff
@@ -52,9 +54,9 @@ void DoPostUninstallOperations(const base::Version& version,
   DCHECK_EQ(survey_url.find(L'?', pos + 1), std::wstring::npos);
   DCHECK_NE(survey_url.back(), L'&');
 #endif
-  auto url = base::StringPrintf(L"%ls&crversion=%ls&os=%ls", survey_url.c_str(),
-                                base::ASCIIToWide(version.GetString()).c_str(),
-                                os_version.c_str());
+  auto url = base::StrCat({survey_url.c_str(), L"&crversion=",
+                           base::ASCIIToWide(version.GetString()).c_str(),
+                           L"&os=", os_version.c_str()});
```

Comment:
`.c_str()` calls should be removed here.

Reply:
Thanks, fixed here f276b8175d61c800ed277c42c090a950359a5d54.